context-free.md

```diff
@@ -1,22 +1,23 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for anyone interested in art and context free grammars.
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to generate recuring shapes.
+This is easy because the language has an intuitive form for recursion.
 
 ## What is hard to do in this language? Why is it hard?
-
+It is difficult to write out words and letters because the programmer has to figure out how each of the letters come together and then implement the shape. 
```

Comment:
Probably the lines and curves, although there is still the issue of playing around until the size of the lines is correct (since we rely on the size limit to break recursion)